### ISSUE TITLE
Apply offset to Gihyeok's data

### ISSUE THREAD

**Issue by danielballan** [AI-multimodal/aimmdb#60]

**Comment by matthewcarbone** [comment]
@danielballan `L3` edge just to be clear! 👌

**Comment by matthewcarbone** [comment]
Note to my knowledge there are no other issues with any other data that require this kind of shift, just FYI.

**Comment by jmaruland** [comment]
Yes, Gihyeok was clear on his instructions about this. I have received the offset values and the indications of which sample name each offset value applies to. Also, in that same email, he supplied the remaining files for 811 Ni L3 data that were missing in the first batch

**Comment by danielballan** [comment]
Sorry, typo. Corrected original post now; thank you.
